collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+priority = 25
+update_every = 1
+
+ORDER = [
+    'alarms',
+]
+
+CHARTS = {
+    'alarms': {
+        'options': ['alarms', 'Alarms', 'status', 'alarms', 'alarms.status', 'line'],
+        'lines': []
+    }
+}
+
+
+class Service(UrlService):
+    def __init__(self, configuration=None, name=None):
+        UrlService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+
+    def validate_charts(self, name, data, algorithm='absolute', multiplier=1, divisor=1):
+        for dim in data:
+            if name not in self.charts:
+                chart_params = [name] + CHARTS[name]['options']
+                self.charts.add_chart(params=chart_params)
+            if dim not in self.charts[name]:
+                self.charts[name].add_dimension([dim, dim, algorithm, multiplier, divisor])
+
+    def _get_data(self):
+        raw_data = loads(self._get_raw_data())
+        alarms = raw_data.get('alarms', {})
+        status_map = {'UNINITIALIZED': 0 , 'UNDEFINED': 0, 'CLEAR': 0, 'WARNING': 1, 'CRITICAL': 2}
```

Comment:
how about make this status_map global var?

Reply:
done